test/service/test_manifest.py

```diff
@@ -1537,7 +1537,12 @@ def test_manifest(self,
         Verify the response from manifest endpoints for all manifest formats
         """
 
-        def test(*, format: ManifestFormat, fetch: bool, url: Optional[furl] = None):
+        def test(*,
+                 format: ManifestFormat,
+                 fetch: bool,
+                 curl: bool = False,
+                 url: furl | None = None):
+            assert not fetch or not curl
```

Comment:
```suggestion
            assert not (fetch and curl)
```

is shorter and much more comprehensible